enhancements/update/available-update-metadata.md

```diff
@@ -0,0 +1,272 @@
+But that property was optional before and did not have clear semantics in either location, so the removal should break no consumers.
+
+### Upgrade / Downgrade Strategy
+
+The YAML rendering of the old and updated type are compatible, with the only difference being optional properties, so there is no need for upgrade/downgrade logic.
+
+### Version Skew Strategy
+
+The YAML rendering of the old and updated type are compatible, with the only difference being optional properties, so there is no need for upgrade/downgrade logic.
+
+## Implementation History
+
+* [API pull request][api-pull-request].
+
+## Drawbacks
+
+A risk in exposing metadata is that users would rely on unstable metadata properties not defined in this enhancement, and then break when Cincinnati removed the property.
+The instability of those properties is clearly explained in this enhancement, but that doesn't mean that users will be happy if they decide to take a risk on an unstable metadata property that is subsequently removed.
+
+## Alternatives
+
+Components like the web console could bypass ClusterVersion and talk to the upstream Cincinnati directly, but this would require significant code duplication.
+
+### Not distinguishing based on data source
+
+Initial versions of this enhancement proposal recommended a single `metadata` property instead of the split `metadata` and `additionalMetadata`.
+But there was [concern][upstream-trust] about the relability of data retrieved from the upstream Cincinnati service.
+The `additionalMetadata` property name was [suggested][additional-metadata-suggestion] was an attempt to address that concern by giving the upstream-sourced data a less authoritative name.
+And `additionalMetadata` without a primary `metadata` property seemed to [beg the "in addition to what?" question][additional-metadata-alone].
+
+The benefit to the `additionalMetadata` demotion is that it gives us space to be explicit about the chain of trust behind the asserted metadata.
+The drawback is that now we have to be explicit about chain of trust in the API documentation, and it's unlikely that many consumers care about chain-of-trust distinctions for `url` or `io.openshift.upgrades.graph.release.channels`.
+It is possible that future well-known metadata properties are more sensitive than [the ones documented by this enhancement](#metadata-properties).
+
+### Capturing this information in a separate custom resource
+
+Instead of changing the `ClusterVersionStatus` properties, we could deprecate them and make `Release` a stand-alone custom resource.
+`Version` would become `ObjectMeta.Name`, and `Image` and `Metadata` would be `Status` properties.
+The benefit of this approach would be a smaller, tighter ClusterVersion.
+The drawback is that you need some way to figure out which is the current release and which are the available updates, which is a bit racier if you can no longer retrieve a single, atomic snapshot with a `GET ClusterVersion`.
```

Comment:
I don't see why that's incompatible with the current proposal.  You would just do both - one is a "these are the available versions" (standalone resource), and here's "a summary of the ones that we think you should upgrade to" inside of cluster version.  We can do the latter before the former.

Reply:
> You would just do both - one is a "these are the available versions" (standalone resource), and here's "a summary of the ones that we think you should upgrade to" inside of cluster version.

What would the standalone available versions be available for, besides "upgrading too" or "the version you're currently reconciling towards"?  If those both stay under ClusterVersion, it feels strange to me to also have stand-alone resources that go into more detail about the releases.  Because once you resolve the referencing issue, what is the benefit to keeping anything more than reference pointers in ClusterVersion itself?

Reply:
Simplicity for clients.  The full set of info you want might be pretty rich as an admin, but for an end user trying to decide might be pretty simple / different.  I think these are clearly two structs - it’s just a matter of only adding fields that help people make the right decision.  You can do a simpler structure and expand later.